org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -477,6 +487,13 @@ package lang {
 
 		override method simplifiedToSmartString(){ return self.stringValue() }
 		override method internalToSmartString(alreadyShown) { return self.stringValue() }
+		method between(min, max) { return (self >= min) && (self <= max) }
+		method squareRoot() { return self ** 0.5 }
+		method square() { return self * self }
+		method even() { return self % 2 == 0 }
+		method odd() { return self.even().negate() }
+		method rem(other) { return self % other }
```

Comment:
Nice to have a `rem` message. I'd just replace the `%`  message with `rem` instead of making it an alias, but this is another discussion we all have to have.